### PR TITLE
EncoderFdkAac: Fix broken detection on Ubuntu 18.04 (lp1919073)

### PR DESCRIPTION
**Pull request by Holzhaus**
Fixes https://bugs.launchpad.net/mixxx/+bug/1919073.

This works for me in an Ubuntu 18.04 LTS VM. Please test.

### CODE REVIEW

**Comment by Be-ing** [comment]
https://doc.qt.io/qt-5/qlibrary.html

> If the file cannot be found, QLibrary tries the name with different platform-specific file prefixes, like "lib" on Unix and Mac

I don't know why that isn't working, but if this one line change works, whatever, let's merge it.

**Comment by Holzhaus** [comment]
Hmm, weird. Now it doesn't work anymore.

**Comment by Holzhaus** [comment]
Look like we actually have to hardcode the filename. No idea why. @daschuer please have a look.

**Comment by Holzhaus** [comment]
For the record: On Arch Linux it works without the workaround.

```
$ pacman -Ql libfdk-aac  | grep so
libfdk-aac /usr/lib/libfdk-aac.so
libfdk-aac /usr/lib/libfdk-aac.so.2
libfdk-aac /usr/lib/libfdk-aac.so.2.0.1
```

File: src/encoder/encoderfdkaac.cpp [hunk 1]
```diff
@@ -76,8 +76,18 @@ EncoderFdkAac::EncoderFdkAac(EncoderCallback* pCallback)
...
+    // Although the line above should suffice, detection of the fdk-aac library
+    // does not work on Ubuntu 18.04 LTS and Ubuntu 20.04 LTS:
+    //
+    //     $ dpkg -L libfdk-aac1 | grep so
+    //     /usr/lib/x86_64-linux-gnu/libfdk-aac.so.1.0.0
+    //     /usr/lib/x86_64-linux-gnu/libfdk-aac.so.1
+    //
+    // As a workaround, we have to hardcode the filenames here.
+    libnames << QStringLiteral("libfdk-aac.so.1")
+             << QStringLiteral("libfdk-aac.so.2");
```
Comment (Be-ing) [inline src/encoder/encoderfdkaac.cpp:88]: Are you sure we need both of these? Is `libfdk-aac` not sufficient?

Comment (Holzhaus) [reply src/encoder/encoderfdkaac.cpp:88]: No it isn't. No idea why.

**Review by Be-ing — APPROVED** [review]

File: src/encoder/encoderfdkaac.cpp (hunk 1 above)
Comment (Holzhaus) [reply src/encoder/encoderfdkaac.cpp:88]: Since we support both version 1 and 2, we need both IMHO.

**Review by daschuer — APPROVED** [review]
LGTM

**Pull request merged**